__tests__/util/index.js

```diff
@@ -114,9 +114,9 @@ async function npmInstall (pathToRunInstallIn) {
   )
 }
 
-async function startPrototype (prototypePath) {
+async function startPrototype (prototypePath, command = 'npm run dev') {
   return exec(
-    'npm run dev',
+    command,
```

Comment:
I think it would be more helpful it we passed this function a value `env` that was either `development` or `production`, and then it would run the right command for us. It could then also set the environment variables that we need (like USE_HTTPS=false).